### ISSUE TITLE
Accessibility issue: Provide valid, concise, and meaningful alternative text for image buttons

### ISSUE THREAD

**Issue by karentsao** [ccnmtl/mediathread#5159]
**Module**:
Collection Detail: Image Type

**Violation**:
Provide valid, concise, and meaningful alternative text for image buttons

**Description**:
[Issue]
There are image buttons that do not provide an accessible name. The zoom controls have a text of + or - which does not convey their purpose.

[User Impact]
Without meaningful alternative text, any information conveyed by these image buttons will not be conveyed to screen reader users and they will be announced as unlabeled buttons.

[Code Reference]
<button title=""Zoom in"" type=""button"" class=""ol-zoom-in"">+</button>

*Major Sheet Row 12 in https://docs.google.com/spreadsheets/d/1L9mwZ55UNKOUOVC_w-fuUjpbeNTyXVE5/edit#gid=512445576

**Comment by nikolas** [comment]
Okay, so after some more research, it looks like the template code in question here is actually present in the OpenLayers pulled in by our package.json: `"ol": "8.2.0"`

So, updating this code will not be so straightforward. It sounds like all we need to do is add an attribute like `aria-label="Zoom in"` to these button elements.

There is some discussion on this here: https://github.com/openlayers/openlayers/issues/12485, as well as adding aria-label to OpenLayers' `<canvas>` element: #5160

**Comment by nikolas** [comment]
https://github.com/ccnmtl/mediathread/pull/5219
